packages/xstate-fsm/test/fsm.test.ts

```diff
@@ -462,4 +390,150 @@ describe('interpreter', () => {
 
     expect(nextState.actions.map((a) => a.type)).toEqual(['action']);
   });
+
+  describe('`start` method', () => {
```

Comment:
this block has just been moved to the bottom as tests were sort of mixed and the order in the file didn't match the order in the test report